e2e/test/iothub/CombinedClientOperationsPoolAmqpTests.cs

```diff
@@ -33,7 +33,8 @@ public async Task DeviceSak_DeviceCombinedClientOperations_MultipleConnections_A
                 .ConfigureAwait(false);
         }
 
-        [LoggedTestMethod, Timeout(TestTimeoutMilliseconds)]
+        [LoggedTestMethod, Timeout(LongRunningTestTimeoutMilliseconds)]
+        [TestCategory("LongRunning")]
```

Comment:
These tests kept timing out at 3mins intermittently. I increased their timeout to 5mins and marked them "LongRunning".